src/core/reaction_methods/tests/ReactionAlgorithm_test.cpp

```diff
@@ -26,12 +26,12 @@
 #include <boost/test/unit_test.hpp>
 
 #include "reaction_methods/ReactionAlgorithm.hpp"
+#include "reaction_methods/ReactionEnsemble.hpp"
```

Comment:
The `ReactionEnsemble` header cannot be included it this file, because it's a unit test for the `ReactionAlgorithm` class. Please either rewrite the new test logic to use the `ReactionAlgorithm` class, or move it to the `ReactionEnsemble_test.cpp` file.